scripts/utils/FileUtils.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Get the line and column from an index in a source string.
+ * @param source - The source string.
+ * @param index - The index in the source string.
+ * @returns The line and column.
+ */
+function getLineAndColumnFromIndex(source: string, index: number): {line: number; column: number} {
+    const substring = source.slice(0, index);
+    const line = substring.split('\n').length;
+    const lastLineBreakIndex = substring.lastIndexOf('\n');
+    const column = lastLineBreakIndex === -1 ? index + 1 : index - lastLineBreakIndex;
+    return {line, column};
+}
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
Definitely NAB but I always prefer using default exports so that it can be namespaced in the import:

```
FileUtils.getLineAndColumnFromIndex
```

that makes it more readable as it's clear it's an external function

Reply:
Ahh yea, sorry i though we stopped doing that, but we only stoped using namespace imports like `import * as SomeUtils from "@utils/SomeUtils"`

Refactored!